File: javascript/src/fibonacci/fibonacci.ts
export const fibonacci = (n: number): number => {
  if (n > 2) {
    return fibonacci(n - 1) + fibonacci(n - 2);
  } else if (n === 1 || n === 2) {
    return 1;
  } else {
    console.log("Do not enter negative values.");

    return 0;
  }
};

File: python/src/fibonacci/fibonacci.py
from functools import lru_cache


@lru_cache(maxsize=5)
def fibonacci(n):
    try:
        if n > 2:
            return fibonacci(n - 1) + fibonacci(n - 2)
        elif n == 1 or n == 2:
            return 1
        else:
            print("Do not enter negative values.")
    except TypeError:
        print("Please enter a number.")
